index.bs

```diff
@@ -3423,6 +3423,33 @@ examples that violate the above rules are {{XMLHttpRequest}} and
 initialisms, even if they are repeated.
 </div>
 
+<h4 id="factory-naming" class="no-num no-toc">Start factory method names with `create` or `from`</h4>
+
+When defining a new factory method, prefix its name with either `create` or `from`.
```

Comment:
There are also many cases where `create()` or `from()` is not a prefix but the entire name of the function, e.g. `Object.create()`, `Temporal.from()` etc. We should probably acknowledge them.

Reply:
See suggestion below.